docs/formatV3.txt

```diff
@@ -51,7 +51,7 @@ version 3 file. This tag has no cryptographic value.
 2.3 P' is the "stretched key" generated from the user's passphrase and
 the SALT, as defined by the hash-function-based key stretching
 algorithm in [KEYSTRETCH] (Section 4.1), with SHA-256 [SHA256] as the
-hash function, and ITER iterations (at least 2048, i.e., t = 11).
+hash function, and ITER iterations (at least 16,348).
```

Comment:
I think 16,348 is much too low. NIST is recommending at least 600,000. I have tested this on several devices (including an older Android device), even with 1,000,000 I don't notice any delay on unlocking the safe.

Can we increase the bare minimum to 2^19 = 524,288 ?

As this change the V3 doc, it might be a good idea to increase the minor version and leave a historic note about the 2,048 minimum. That way, future implementors know what to expect from existing safes.

Reply:
To illustrate this, below is a dialog that will become available on iOS in the near future for advanced users, the screenshot is taken on my iPhone Pro 15. After accepting the recommended value the safe still opens with unnoticeable delay.
<img width="299" alt="Screenshot 2025-01-22 at 16 04 51" src="https://github.com/user-attachments/assets/a9551a15-8962-4194-ad72-31f490033adb" />